ExtractArubaSW.ps1

```diff
@@ -0,0 +1,683 @@
+﻿
+ <#
+  .SYNOPSIS
+  Ce script permet d'extraire des données d'un switch Aruba et de classer ces informations d'un tableau Excel
```

Comment:
Can you translate to english your comment ?

Reply:
done